agent/config/runtime.go

```diff
@@ -22,6 +22,14 @@ type RuntimeSOAConfig struct {
 	Minttl  uint32 // 0,
 }
 
+// Cache represent the cache configuration of agent
+type Cache struct {
+	// EntryFetchMaxBurst max burst size of RateLimit for a single cache entry
+	EntryFetchMaxBurst int
```

Comment:
same as above, I think we can remove

Reply:
What do you think about my comment https://github.com/hashicorp/consul/pull/8226#discussion_r459112485 ?